staging/knative/charts/eventing/crds/eventing-crds.yaml

```diff
@@ -1390,10 +1390,10 @@ spec:
           type: string
           jsonPath: ".status.conditions[?(@.type=='Ready')].reason"
     - !!merge <<: *version
```

Comment:
looking at the upstream file, i don't think this indention is broken there. is something else in our automation doing this?

Reply:
@cbuto I ran my upgrade script multiple times and those were there. I just `curl`'d the file again and the indents are normal. Maybe my laptop is doing it. I feel crazy 🫠

Reply:
I'm leaving it in